### ISSUE TITLE
problem with fetching the Planck map

### ISSUE THREAD

**Issue by bersavosh** [gregreen/dustmaps#18]
When following the [installation](https://dustmaps.readthedocs.io/en/latest/installation.html) procedure I encounter `KeyError: 'content-length'` while trying to fetch the Planck data. I'm using Python 3.6.6 Anaconda on Ubuntu 18.04.1 LTS 64-bit and I installed the package using `pip`.

I tried briefly tweaking the source code based on some [suggestions](https://stackoverflow.com/questions/19255628/python-dictionary-key-error-cannot-resolve) but I did not manage to get it to work. 

Please find details below.
Thanks,
Arash

```
Python 3.6.6 |Anaconda custom (64-bit)| (default, Oct  9 2018, 12:34:16) 
[GCC 7.3.0] on linux
>>> from dustmaps.config import config
>>> config['data_dir'] = './'
>>> import dustmaps.planck
>>> dustmaps.planck.fetch()
Downloading http://pla.esac.esa.int/pla/aio/product-action?MAP.MAP_ID=HFI_CompMap_ThermalDustModel_2048_R1.20.fits ...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../dustmaps/planck.py", line 132, in fetch
    fetch_utils.download_and_verify(url, md5, fname=fname)
  File "/.../dustmaps/fetch_utils.py", line 236, in download_and_verify
    content_length = int(r.headers['content-length'])
  File "/.../requests/structures.py", line 52, in __getitem__
    return self._store[key.lower()][1]
KeyError: 'content-length'
```

**Comment by gregreen** [comment]
@bersavosh, can you try reinstalling dustmaps from the git master branch? I think the particular bug you're encountering was addressed in 4c48e989c893a76f8107bf28d3e2620fd704b6ae, which hasn't made its way into a PiPy release yet.

If that commit fixes the bug you're encountering, I'll close this issue and get the commit into a new PiPy release.

**Comment by bersavosh** [comment]
@gregreen, thanks for the response. I did as you said and the issue is resolved.

**Comment by gregreen** [comment]
The commit that resolves this issue, 4c48e98, is included in the latest release (v1.0.3), which is now available through `pip`.
